src/components/MarkdownProvider/components/Anchor.ts

```diff
@@ -5,22 +5,22 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-import styled, { DefaultTheme } from 'styled-components';
-import { getColor, mediaQuery } from '@zendeskgarden/react-theming';
+import styled from 'styled-components';
+import { SELECTOR_FOCUS_VISIBLE, focusStyles, mediaQuery } from '@zendeskgarden/react-theming';
 import { Anchor } from '@zendeskgarden/react-buttons';
-import { math, rgba } from 'polished';
-
-const boxShadow = (theme: DefaultTheme) => {
-  const color = getColor('primaryHue', 600, theme);
-  const shadow = rgba(color as string, 0.35);
-
-  return `inset ${theme.shadows.sm(shadow)}`;
-};
+import { math } from 'polished';
 
 export const StyledAnchor = styled(Anchor).attrs(props => ({
   /* eslint-disable-next-line prefer-named-capture-group */
   isExternal: props.href && !/^(#|\/(?!\/))/u.test(props.href)
 }))`
+  ${({ theme }) =>
+    focusStyles({
+      theme,
+      inset: true,
+      styles: { outlineColor: 'transparent' }
+    })}
```

Comment:
Shouldn't this only apply to the `&.anchor.before` so that only the heading link icons are affected?